bitnami/testlink/values.yaml

```diff
@@ -71,6 +71,18 @@ allowEmptyPassword: "yes"
 ## External database configuration
 ##
 externalDatabase:
+
+  ##If set to 'true' db and db user will be created as specified
+  create: false
+  
+  ## Database user privileges
+#  privileges: ALL
+
+  ##admin user to create db and user.
+#  root:
+#    user:
+#    password:
```

Comment:
Our automated testing system is failing with this structure. Do you mind changing it to rootUser and rootPassword?

Reply:
No problem, changes done.